SPECS/calamares/calamares.spec

```diff
@@ -7,19 +7,19 @@ Summary:        Installer from a live CD/DVD/USB to disk
 # https://github.com/calamares/calamares/issues/1051
 Name:           calamares
 Version:        3.3.1
-Release:        4%{?dist}
+Release:        5%{?dist}
 License:        GPLv3+
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://calamares.io/
 # Source0..19 - source tarballs
 Source0:        https://github.com/calamares/calamares/releases/download/v%{version}/%{name}-%{version}.tar.gz
 # Source1..4 is an artifact from https://dev.azure.com/mariner-org/mariner/_git/calamares-installer-module
-Source1:        calamares-users-3.0.1.tar.gz
-Source2:        calamares-finished-3.0.1.tar.gz
-Source3:        calamares-welcome-3.0.1.tar.gz
-Source4:        calamares-partition-3.0.1.tar.gz
-Source5:        calamares-license-3.0.1.tar.gz
+Source1:        calamares-users-3.0.3.tar.gz
```

Comment:
why don't these versions match the package? (3.3.1)?

Reply:
These versions are come from our internal implementation/adaptation of the calamares modules. As a result, we decided to name them based on the Azure Linux version. In the past, 1.0 and 2.0 shared the same modules and simply used the 1.X.Y nomenclature.